azure-pipelines.yml

```diff
@@ -0,0 +1,52 @@
+jobs:
+  - job: test
+    displayName: Test
+    pool:
+      vmImage: vs2017-win2016
+    steps:
+      - task: NodeTool@0
+        inputs:
+          versionSpec: '12.x'
+        displayName: 'Install Node.js'
+      - powershell: | # Not that cool, but it does the job.
```

Comment:
```suggestion
      - powershell: |
```

Reply:
Don't pick a fight with windows users. There are a surprising number of them.

Reply:
It's not a fight with the windows users, it's a fight with myself unable to find a better solution. I'll remove it though.